openapi.yml

```diff
@@ -0,0 +1,1855 @@
+openapi: 3.0.3
```

Comment:
ok, so now the idea is to commit this file to the repo, right? I understand this file doesn't go through any post processing and is persisted to the repo as is from the repo upstream, right?